dev/devicelab/lib/tasks/run_tests.dart

```diff
@@ -188,25 +185,6 @@ class WindowsRunOutputTest extends DesktopRunOutputTest {
       _buildOutput.hasMatch,
       'Building Windows application...',
     );
-
-    final String buildMode = release ? 'Release' : 'Debug';
-    _findNextMatcherInList(
-      stdout,
-      (String line) {
-        if (!_builtOutput.hasMatch(line) || !line.contains(buildMode)) {
-          return false;
-        }
```

Comment:
Shouldn't we keep this? The size information was removed, but we still output the exe's path

Reply:
Ah, it looks like this change makes `flutter run -d windows` not output the .exe path. However, `flutter run` on an Android device does output the .apk path. We should be consistent and continue to output the .exe path on Windows.

Reply:
I updated the PR to print when running as well. I opted to not output the size when the output isn't self-contained (Windows, Linux, web) as the the information is incomplete and misleading (e.g. a ~0.1MB .exe)